crates/aptos/e2e/cases/account.py

```diff
@@ -80,3 +82,48 @@ def test_account_lookup_address(run_helper: RunHelper, test_name=None):
         raise TestError(
             f"lookup-address result does not match {run_helper.get_account_info().account_address}"
         )
+
+
+@test_case
+def test_account_resource_account(run_helper: RunHelper, test_name=None):
```

Comment:
Could you make a call to the local testnet to confirm that the resource account was actually created on chain?